### DISCUSSION TITLE
repetitive inclusion of mcu type to the build command

### DISCUSSION THREAD

**Discussion by cemox** [Alex079/vscode-avr-helper#46] [General]
Hello, thank you for this wonderful plugin.

After every build, mcu type is added to the command repetitively, but it compiles without any issues

`Command: /usr/bin/avr-size -C --mcu=atmega8 --mcu=atmega8 --mcu=atmega8 --mcu=atmega8 /home/cemo/AVRProjects/Atmega8_Oled128x32/build/output.elf`

notice that **--mcu=atmega8** is added to the command four times

Am I doing something wrong?

**Comment by Alex079** [comment]
Hello!

This is a bug. Thank you for reporting it. I am working on a fix.

**Reply by Alex079** [reply]
#47 should fix it

**Comment by cemox** [comment]
thank you, I will check it out

Alex079 ***@***.***>, 5 Nis 2023 Çar, 23:23 tarihinde şunu
yazdı:
